src/ApplicationManager.cpp

```diff
@@ -44,6 +45,7 @@ void ApplicationManager::start() {
     plays.emplace_back(std::make_unique<rtt::ai::stp::play::Defend>());
     plays.emplace_back(std::make_unique<rtt::ai::stp::play::DefensiveFormation>());
     plays.emplace_back(std::make_unique<rtt::ai::stp::play::AggressiveFormation>());
+    plays.emplace_back(std::make_unique<rtt::ai::stp::play::BallPlacement>());
     playChecker.setPlays(plays);
```

Comment:
Why are these unique pointers again when they're owned by `plays` and you can just pass Play* instead?...